File: samples/client/petstore/java/src/main/java/org/openapijsonschematools/client/paths/fakerefsobjectmodelwithrefprops/post/requestbody/content/applicationjson/ApplicationjsonSchema.java
package org.openapijsonschematools.client.paths.fakerefsobjectmodelwithrefprops.post.requestbody.content.applicationjson;

import org.checkerframework.checker.nullness.qual.Nullable;
import org.openapijsonschematools.client.components.schemas.ObjectModelWithRefProps;

public class ApplicationjsonSchema extends ObjectModelWithRefProps {
    // $refed class
    
    
    public static class ApplicationjsonSchema1 extends ObjectModelWithRefProps1 {
        private static @Nullable ApplicationjsonSchema1 instance = null;
        public static ApplicationjsonSchema1 getInstance() {
            if (instance == null) {
                instance = new ApplicationjsonSchema1();
            }
            return instance;
        }
    }
}

File: samples/client/petstore/kotlin/src/main/kotlin/org/openapijsonschematools/client/paths/fakebodywithfileschema/put/requestbody/content/applicationjson/ApplicationjsonSchema.kt
package org.openapijsonschematools.client.paths.fakebodywithfileschema.put.requestbody.content.applicationjson;

import org.checkerframework.checker.nullness.qual.Nullable;
import org.openapijsonschematools.client.components.schemas.FileSchemaTestClass;

public class ApplicationjsonSchema extends FileSchemaTestClass {
    // $refed class
    
    
    public static class ApplicationjsonSchema1 extends FileSchemaTestClass1 {
        private static @Nullable ApplicationjsonSchema1 instance = null;
        public static ApplicationjsonSchema1 getInstance() {
            if (instance == null) {
                instance = new ApplicationjsonSchema1();
            }
            return instance;
        }
    }
}
